src/features/joinForms/components/JoinSubmissionTable.tsx

```diff
@@ -0,0 +1,124 @@
+  const classes = useStyles();
+  const messages = useMessages(messageIds);
+  const theme = useTheme();
+  const { approveSubmission } = useJoinSubmissionMutations(orgId);
+
+  return (
+    <Box bgcolor={theme.palette.background.paper} m={2}>
+      <DataGridPro
+        columns={[
+          {
+            disableColumnMenu: true,
+            field: 'state',
+            flex: 1,
+            headerName: messages.status(),
+            renderCell: (params) => {
+              return (
+                <Box className={classes.chip}>
+                  {messages.states[params.row.state]()}
+                </Box>
+              );
+              // />
+            },
+            valueGetter: (params) => params.row.state,
+          },
+          {
+            disableColumnMenu: true,
+            field: 'first_name',
+            flex: 1,
+            headerName: messages.submissionList.firstName(),
+            valueGetter: (params) => params.row.person_data.first_name,
+          },
+          {
+            disableColumnMenu: true,
+            field: 'last_name',
+            flex: 1,
+            headerName: messages.submissionList.lastName(),
+            valueGetter: (params) => params.row.person_data.last_name,
+          },
+          {
+            disableColumnMenu: true,
+            field: 'form',
+            flex: 1,
+            headerName: messages.submissionList.form(),
+            valueGetter: (params) => params.row.form.title,
+          },
+          {
+            disableColumnMenu: true,
+            field: 'submitted',
+            flex: 1,
+            headerName: messages.submissionList.timestamp(),
+            type: 'dateTime',
+            valueGetter: (params) => new Date(params.row.submitted),
+          },
+          {
+            align: 'right',
+            disableColumnMenu: true,
+            field: 'actions',
+            flex: 1,
+            headerName: '',
+            renderCell: (params) => {
+              if (params.row.state !== 'accepted') {
+                return (
+                  <Box display="flex" gap={2}>
+                    {/* TODO: Handle rejectButton click */}
+                    {/* <Button variant="text">
+                      <Msg id={messageIds.submissionList.rejectButton} />
+                    </Button> */}
+                    <Button
+                      onClick={(event) => {
+                        approveSubmission(params.row.id);
+                        event.stopPropagation();
+                      }}
+                      variant="outlined"
+                    >
+                      <Msg id={messageIds.submissionList.approveButton} />
+                    </Button>
+                  </Box>
+                );
+              } else {
+                return null;
+              }
+            },
+            sortable: false,
+          },
+        ]}
+        hideFooter
+        isRowSelectable={() => false}
```

Comment:
I believe that is how `DataGridPro` works. This property is supposed to be a function, and that function will be called for every row using the row as a parameter. Because we don't care about the parameter here (always returning `false`) we also do not accept a parameter, but that doesn't mean we could replace it with a constant, because that's not how the component expects the property to work.